src/docs/Components/Buttons/index.mdx

```diff
@@ -11,80 +11,81 @@ status:
 
 ## Variations
 
-### Four color schemes for each type of action
+### Color themes
 
-You can use **primary**, **neutral**, & **danger** color scheme depending of the type of action the button trigger.
-There is also an alternative primary color scheme, only used for Campus : **primary-02**.
+We created different color themes to suit what you need in your design.
+You can use `primary` (default), `danger` and `neutral` theme in `solid` and `bordered` declination.
+There is also an alternative primary color, only used for Campus: `primary-02`.
 
 <Preview path="button-styles" nude />
-
 <br />
 
-| Color scheme | Usage                                                                                     |
-| ------------ | ----------------------------------------------------------------------------------------- |
-| Primary      | For main actions on the page or engaging actions                                          |
-| Primary-02   | For main specific actions dedicated to Campus                                             |
-| Neutral      | For secondary or minor actions on the page                                                |
-| Danger       | For buttons who the following actions is irrevocable, like deleting a product in a basket |
+| Color scheme | Usage                                                                                   |
+| ------------ | --------------------------------------------------------------------------------------- |
+| Primary      | For main actions on the page or engaging actions                                        |
+| Primary-02   | For main specific actions dedicated to Campus                                           |
+| Neutral      | For secondary or minor actions on the page                                              |
+| Danger       | For buttons who the following actions is irrevocable, like deleting a product in a cart |
 
-### Three sizes to create hierarchy
+### Sizes
 
-There is **3 sizes** for the buttons to create hierarchy and for a better adaptability :
+There is 3 sizes for the buttons to create hierarchy and for a better adaptability.
 
-- Small
-- Medium
-- Large
+| Size                   | Value        |
+| ---------------------- | ------------ |
+| **Small**              | 2mu (32px)   |
+| **Medium** (_default_) | 3mu (48px)   |
+| **Large**              | 3.5mu (56px) |
 
 <Preview path="button-sizes" nude />
-
 <br />
 
-### A button width can either fit its content or its container
+### Styles
```

Comment:
still here...

Reply:
`@if map-get($sizeProps, 'icon-size') != map-get($button-medium, 'icon-size') {`

Reply:
but I think we will need those values if we use responsive modifiers right ?

Reply:
@caqueste @nherwegh Can you take a look at it?

Reply:
let's forget about it

Reply:
@gael-boyenval we fix it with the designers, can you take a look ? Thanks.